feat: added log coloring with chalk

tonioriol opened the pull request.

lalmqvist commented:
![image](https://user-images.githubusercontent.com/34321514/92239235-f29e2680-eeba-11ea-9a0a-e13de976e369.png)
I get this error when testing this branch locally. But maybe it's not related to your changes?

tonioriol commented:
> ![image](https://user-images.githubusercontent.com/34321514/92239235-f29e2680-eeba-11ea-9a0a-e13de976e369.png)
> I get this error when testing this branch locally. But maybe it's not related to your changes?

it wasnt but i fixed it on master and rebased si it should be fixed
